packages/in_app_purchase/lib/src/in_app_purchase_connection/google_play_connection.dart

```diff
@@ -59,4 +60,33 @@ class GooglePlayConnection
       _billingClient.startConnection(onBillingServiceDisconnected: () {});
 
   Future<void> _disconnect() => _billingClient.endConnection();
+
+  /// Query the product detail list.
+  ///
+  /// This method only returns [QueryProductDetailsResponse].
+  /// To get detailed Google Play sku list, use [BillingClient.querySkuDetails]
+  /// to get the [SkuDetailsResponseWrapper].
+  Future<QueryProductDetailsResponse> queryProductDetails(
+      Set<String> identifiers) async {
+    SkuDetailsResponseWrapper inappResponse =
+        await _billingClient.querySkuDetails(
```

Comment:
nit: It would be a little faster if these two queries happened in parallel and we blocked on them both completing in general, instead of blocking on both of them sequentially.

I think you can use [`Future.wait`](https://api.dartlang.org/stable/2.1.0/dart-async/Future/wait.html) to block on multiple futures resolving.